engine/runtime-instrument-common/src/main/java/org/enso/interpreter/service/ExecutionService.java

```diff
@@ -306,6 +306,7 @@ private FunctionCallInstrumentationNode.FunctionCall substituteMissingArguments(
    * @return a result of evaluation
    */
   public Object evaluateExpression(Module module, String expression) {
+    LOGGER.debug("evaluateExpression in {} code: {}", module.getName(), expression);
```

Comment:
I also think this should be `trace`

Reply:
Be it trace - dd11b4a4d20dc484db09825f7a26e281b9fd3454